Earthfile

```diff
@@ -329,7 +336,17 @@ build-framework-image:
 
     COPY (+framework/framework --SECURITY_PROFILE=$_SECUIRTY_PROFILE) /
 
-    SAVE IMAGE --push $IMAGE_REPOSITORY_ORG/framework:${VERSION}_${_SECUIRTY_PROFILE}
+    SAVE IMAGE --push $IMAGE_REPOSITORY_ORG/framework:${FRAMEWORK_VERSION}_${_SECUIRTY_PROFILE}
+
+kairos-dockerfile:
+    ARG --required FAMILY
+    COPY ./images .
+    RUN --no-cache cat <(echo "# This file is auto-generated with the command: earthly +kairos-dockerfile --FAMILY=${FAMILY}") \
+        Dockerfile.$FAMILY \
+        <(echo) \
+        <(sed -n '/# WARNING:/!p' Dockerfile.kairos) \
+        > ./Dockerfile
+    SAVE ARTIFACT Dockerfile AS LOCAL images/Dockerfile.kairos-${FAMILY}
```

Comment:
Should these files be checked in git? Otherwise gitignore?

Reply:
I was hesitating about this but I think you're right they should since that's what the end user will pick